platform_plugin_ontask/data_summary/backends/completion.py

```diff
@@ -53,55 +49,46 @@ class CompletionDataSummary(DataSummary):
             "0": "course-v1:edunext+ontask+demo",
             "1": "course-v1:edunext+ontask+demo",
         },
-        "block_id": {
-            "0": "9c56dbeb30504c8fb799553f080cf15d",
-            "1": "9c56dbeb30504c8fb799553f080cf15d",
-        },
-        "block_name": {
+        "unit_9c56dbeb30504c8fb799553f080cf15d_name": {
             "0": "Unit 1.1",
             "1": "Unit 1.1",
         },
-        "completed": {
+        "unit_9c56dbeb30504c8fb799553f080cf15d_completed": {
             "0": False,
             "1": True,
         }
     }
+
     ```
     """
 
-    ID = "id"
-    USER_ID = "user_id"
     EMAIL = "email"
     USERNAME = "username"
     COURSE_ID = "course_id"
-    UNIT_ID = "unit_id"
-    UNIT_NAME = "unit_name"
-    COMPLETED = "completed"
+    UNIT_NAME = "unit_{}_name"
+    COMPLETED = "unit_{}_completed"
```

Comment:
```suggestion
    UNIT_COLUMN_NAME = "unit_{}_name"
    COMPLETED_COLUMN_NAME = "unit_{}_completed"
```

Reply:
And so on.

Reply:
Done: https://github.com/eduNEXT/platform-plugin-ontask/commit/1b3fe5c5c597068413a555edf47ff722223e2b02